### ISSUE TITLE
How to use MigLayout with FXML?

### ISSUE THREAD

**Issue by wimdeblauwe** [mikaelgrev/miglayout#69]
Is there a way to use MigLayout in an FXML file and specify the constraints?

```
<MigPane xmlns="http://javafx.com/javafx"
         xmlns:fx="http://javafx.com/fxml"
         fx:controller="com.company.project.login.LoginScreenController"
>
    <layoutConstraints>
<!-- How to specify constraints here? -->
    </layoutConstraints>
    <rowConstraints>

    </rowConstraints>
    <columnConstraints>

    </columnConstraints>
...
</MigPane>
```

**Comment by tbee** [comment]
Yes there is. The JavaFX version contains some tests showing how to do that: specify MigPane.cc as an attribute.

https://github.com/mikaelgrev/miglayout/blob/master/javafx/src/test/resources/MigPaneTest8.xml

**Comment by wimdeblauwe** [comment]
Thank you! Would be great if this was in the documentation somewhere.

**Comment by wimdeblauwe** [comment]
I just tried the example, but I get errors.

If I do something like:

    <TextField fx:id="usernameField" MigPane.cc="growx, wrap"/>

I get:

    com.sun.javafx.fxml.PropertyNotFoundException: Static property "cc" does not exist or is read-only.

If I do:

    <MigPane xmlns="http://javafx.com/javafx"
         xmlns:fx="http://javafx.com/fxml"
         rows="wrap"
    >

I get:

    com.sun.javafx.fxml.PropertyNotFoundException: Property "rows" does not exist or is read-only.

I am using MigLayout 5.2

**Comment by tbee** [comment]
Make sure the correct version of MigPane is included. There is are two, the one in the fxml subpackage contains extentions needed to be used in fxml.

<?importorg.tbee.javafx.scene.layout.fxml.*?>

**Comment by wimdeblauwe** [comment]
Thanks, I was using the wrong import indeed.
